packages/mui-base/src/useMenu/useMenu.types.ts

```diff
@@ -18,7 +18,7 @@ export interface UseMenuParameters {
   /**
    * Callback fired when the menu is opened or closed.
    */
-  onOpenChange?: (open: boolean) => void;
+  onOpenChange?: (event: React.MouseEvent<Element, Event> | null, open: boolean) => void;
```

Comment:
The event could be a `KeyboardEvent` as well?

Reply:
Or `FocusEvent`, as the menu closes when it loses focus.